src/main/java/wellnus/reflection/command/UnlikeCommand.java

```diff
@@ -170,13 +162,18 @@ public void execute() {
      * A valid index will only be removed(i.e. passed validateCommand()) if the favorite list in not empty.
      *
      * @param questionIndex User input of the index of question to be removed from favorite list.
-     * @throws BadCommandException If there is not a set of question generated yet.
      * @throws TokenizerException If there is error in tokenization of index
      * @throws StorageException If there is error in storing the data
      */
-    public void removeFavQuestion(String questionIndex) throws BadCommandException, TokenizerException,
-            StorageException {
+    public void removeFavQuestion(String questionIndex) throws TokenizerException, StorageException,
+            NumberFormatException, ReflectionException, BadCommandException {
```

Comment:
Hmm I think NumberFormatException should be unnecessary in this case? Since validateCommand() is called before this and ensures that there won't be invalid integers passed into `Integer.parseInt()`